source/css/sass/mixins/_typography.scss

```diff
@@ -109,3 +109,17 @@
 @mixin label-tag-typography() {
   @include _label-typography($color-primary-normal);
 }
+
+@mixin hidden-link($hover-color: $color-primary-dark) {
+  color: inherit;
+  text-decoration: inherit;
+  all: inherit;
+
+  @supports (all: inherit) {
+    cursor: pointer;
```

Comment:
Does this need to be within a feature query?

Reply:
Should be safe not to, but added to clarify why it's there. Can remove if you like.

Reply:
Okay, may as well leave it in :-)